rpc_util.go

```diff
@@ -140,6 +141,7 @@ type callInfo struct {
 	failFast  bool
 	headerMD  metadata.MD
 	trailerMD metadata.MD
+	peer      peer.Peer
```

Comment:
Is it better to use a `*peer.Peer` here?
So that we don't need to copy peer in `recvResponse`.